Casks/g/gridtracker2.rb

```diff
@@ -0,0 +1,28 @@
+cask "gridtracker2" do
+  version "2.241107.2"
+  sha256 "abd3b79192b109f7df2947c36d2ed5f6c4e258c85a7f44592f4c0df0d4ae5f73"
+
+  url "https://download2.gridtracker.org/GridTracker2-#{version}-universal-mac.zip"
+  name "GridTracker2"
+  desc "Warehouse of amateur radio information presented in an easy to use interface"
+  homepage "https://gridtracker.org/"
+
+  livecheck do
+    url "https://download2.gridtracker.org/latest-mac.yml"
+    strategy :electron_builder
+  end
+
+  depends_on macos: ">= :catalina"
+
+  app "GridTracker2.app"
+
+  zap trash: [
+    "~/Library/Application Support/GridTracker",
+    "~/Library/Application Support/GridTracker2",
+    "~/Library/Caches/GridTracker",
+    "~/Library/Preferences/org.gridtracker.gridtracker.plist",
+    "~/Library/Preferences/org.gridtracker.GridTracker2.plist",
+    "~/Library/Saved Application State/org.gridtracker.gridtracker.savedState",
+    "~/Library/Saved Application State/org.gridtracker.GridTracker2.savedState",
```

Comment:
```suggestion
    "~/Library/Application Support/GridTracker*",
    "~/Library/Caches/GridTracker",
    "~/Library/Preferences/org.gridtracker.GridTracker*.plist",
    "~/Library/Saved Application State/org.gridtracker.GridTracker*.savedState",
```

We can combine these entries using a glob `*`.

Reply:
This is cool!